web-api/elasticsearch/searchAll.js

```diff
@@ -0,0 +1,56 @@
+const { get } = require('lodash');
+const { getClient } = require('./client');
+
+const CHUNK_SIZE = 5000;
+
+/**
+ * Uses Elasticsearch's `search_after` to return all results for the provided query
+ *
+ * @param {Object} clientArgs object containing the keys 'environmentName' and 'version'
+ * @param {String} index Elasticsearch index
+ * @param {Object} query Elasticsearch query
+ * @param {Array<Object>} sort array of objects describing the fields and direction to sort by
+ * @param {Array<String>} source array of fields to return
+ * @returns {Promise<Array<Object>>} results
+ */
+const searchAll = async (clientArgs, index, query, sort, source) => {
```

Comment:
I would recommend refactoring this to be an option object instead of an order list of arguments.  Most of the functions in our application take in either 1 or 2 arguments max using objects with named properties